src/components/Account/ReputationView.scss

```diff
@@ -0,0 +1,4 @@
+.reputationAmount {
+	display: inline-block;
```

Comment:
I think for some reason all your .scss files are using tabs instead of 2 spaces, can you check that?